### PR TITLE
Meetup demo: schema defaults

### PR DESCRIPTION
**Pull request by jossmac**
Add defaults to various meetup schemas

### CODE REVIEW

File: demo-projects/meetup/schema.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const cloudinaryAdapter = new CloudinaryAdapter({
 });
 
 const access = {
-  userIsAdmin: ({ authentication: { item: user } }) => !!(user && user.isAdmin),
```
Comment (dcousens) [inline demo-projects/meetup/schema.js]: Why not live dangerously and rely on falsy?

Comment (JedWatson) [reply demo-projects/meetup/schema.js]: Agreed, we don't seem to do this elsewhere..?

Comment (jossmac) [reply demo-projects/meetup/schema.js]: I don't have a horse in this race, just prefer explicit `Boolean` over double bang.

File: demo-projects/meetup/schema.js [hunk 2]
```diff
@@ -48,7 +48,7 @@ exports.User = {
     name: { type: Text },
     email: { type: Text, isUnique: true, access: { read: access.userIsAdminOrPath('id') } },
     password: { type: Password, isRequired: true },
-    isAdmin: { type: Checkbox, access: { update: access.userIsAdmin } },
+    isAdmin: { type: Checkbox, access: { update: access.userIsAdmin }, defaultValue: false },
```
Comment (jossmac) [inline demo-projects/meetup/schema.js]:
```suggestion
    isAdmin: { type: Checkbox, access: { update: access.userIsAdmin } },
```

File: demo-projects/meetup/schema.js [hunk 3]
```diff
@@ -96,7 +96,7 @@ exports.Talk = {
     name: { type: Text },
     event: { type: Relationship, ref: 'Event.talks' },
     speakers: { type: Relationship, ref: 'User.talks', many: true },
-    isLightningTalk: { type: Checkbox },
+    isLightningTalk: { type: Checkbox, defaultValue: false },
```
Comment (jossmac) [inline demo-projects/meetup/schema.js]:
```suggestion
    isLightningTalk: { type: Checkbox },
```

**Pull request merged**